scripts/build.js

```diff
@@ -24,7 +24,7 @@ for (const arg of process.argv) {
 // Run the build
 var buildSystem = new cmake.BuildSystem({
     target: "install",
-    debug: process.argv.includes('--debug'),
+    debug: true,
```

Comment:
Is it neccessary? 🤔 Why do we need to build with debug all the time?

Reply:
Oops, did not mean to include that